### ISSUE TITLE
[Labels] Handle situations where we don't have rate or box packing solution from checkout

### ISSUE THREAD

**Issue by DanReyLop** [Automattic/woocommerce-services#446]
Depends on #445 

If the user selected free shipping or used a non-WCC shipping method, the order metadata won't have packaging info. In that case, before reaching the Packaging step, a packaging request must be made to the server with the items of the shopping cart, and the response must be stored client-side (at least in the JS state).

After that point, the Packaging step should behave exactly like described in #445

**Comment by allendav** [comment]
Moving this to Sprint 5, since that is where we are going to tackle free shipping flow

**Comment by kellychoffman** [comment]
I arrived at this issue when I added a manual order and did not chose a shipping rate. I went to click on the Create Label button and nothing happened. Besides this error:

`index.js:13 Uncaught (in promise) TypeError: Cannot read property 'weight' of undefined(…)`

**Comment by allendav** [comment]
@kellychoffman 's error was coming from the last line of this excerpt from client/shipping-label/views/purchase/steps/packages/index.js

```
const PackagesStep = ( { values, storeOptions, labelActions, errors, expanded } ) => {
	const packageIds = Object.keys( values );
	const firstPackageId = packageIds[ 0 ];
	const firstPackage = values[ firstPackageId ];
	const isValid = 0 < firstPackage.weight;
```

**Comment by allendav** [comment]
@DanReyLop - is this still a problem?

**Comment by DanReyLop** [comment]
Yes, I've just tried it and there are some problems. The PHP side defaults to use "individual packaging" if no packaging data is present, but for some reason with the latest changes it's misinterpreted. Plus, no request to the server is made to run the box-packaging algorithm.

**Comment by allendav** [comment]
See also #759

**Comment by allendav** [comment]
Possibly fixed, at least partially, by #751 ?

**Comment by robobot3000** [comment]
Yeah, this has been partially fixed by #751 and the remaining work is split into the following:
* automatically pack the items: #759 
* server side work for the above: Automattic/woocommerce-connect-server#716
* add a warning for when there are no packages configured: #758

**Comment by v18** [comment]
Related to #503

**Comment by laurendavissmith** [comment]
seems like this is no longer needed given that there are other issues open that cover the remaining pieces here
